docs/source/modin/supported/groupby_supported.rst

```diff
@@ -16,7 +16,7 @@ Indexing, iteration
 +-----------------------------+---------------------------------+----------------------------------------------------+
 | GroupBy method              | Snowpark implemented? (Y/N/P/D) | Notes for current implementation                   |
 +-----------------------------+---------------------------------+----------------------------------------------------+
-| ``get_group``               | N                               |                                                    |
+| ``get_group``               | P                               | Imlemented for DataFrameGroupBy objects only       |
```

Comment:
Based on the NotImplementedError message, can you add more details on the support in this section (i.e. not implemented for Grouper, axis=1, etc.)?

Reply:
This limitation is true for all of groupby, so it would be better to update the docs as a whole. None of the other APIs have this message so it is needed for all of them. I added a ticket to track this, but I don't think it should be updated in this PR: https://snowflakecomputing.atlassian.net/browse/SNOW-1482636

Reply:
Makes sense, thanks!